testdata/project-v3-multigroup/config/crd/bases/ship.testproject.org_cruisers.yaml

```diff
@@ -1,9 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.6.2
```

Comment:
why we are downgrading this one?

Reply:
Temporary to make `make generate` work, details in PR description. Not sure what the fix for the problem I explained is, sure it's something simple that I'm missing.